pkg/coordinator/sweep.go

```diff
@@ -0,0 +1,108 @@
+package coordinator
+
+import (
+	"fmt"
+	"math/big"
+	"regexp"
+	"strconv"
+
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/tbtc"
+)
+
+type btcTransactions = struct {
+	FundingTxHash      bitcoin.Hash
+	FundingOutputIndex uint32
+}
+
+const requiredFundingTxConfirmations = uint(6)
+
+// BitcoinTxRegexp defines a format in which bitcoin transactions are expected
+// to be provided as strings.
+// The format is: <unprefixed transaction hash>:<output index>
+// e.g. bd99d1d0a61fd104925d9b7ac997958aa8af570418b3fde091f7bfc561608865:1:8392394
```

Comment:
Worth mentioning that we expect the transaction hash to be in the `bitcoin.ReversedByteOrder`.

Reply:
f8d884ea468d27c0b59828ee30dfba078178ee13